src/archutils/Win32/CrashHandlerChild.cpp

```diff
@@ -766,35 +765,10 @@ INT_PTR CrashDialog::HandleMessage( UINT msg, WPARAM wParam, LPARAM lParam )
 			EndDialog( hDlg, FALSE );
 			break;
 		case IDC_BUTTON_REPORT:
-			GotoURL( REPORT_BUG_URL );
+			// safe to remove button?
```

Comment:
Yeah I think it would be safe to remove the button but I think that can be in a separate PR. It's not as pressing as getting this PR in.